src/main/java/net/sf/jabref/importer/fileformat/MedlinePlainImporter.java

```diff
@@ -238,11 +237,7 @@ public ParserResult importDatabase(BufferedReader reader) throws IOException {
     }
 
     private boolean checkLineValidity(String line) {
-        if (line.length() >= 4) {
-            return line.contains("-") || (line.charAt(4) != '-') ? true : false;
-        } else {
-            return false;
-        }
+        return (line.length() >= 5) ? line.contains("-") && (line.charAt(4) == '-') : false;
```

Comment:
I think `line.contains("-")` is unnecessary when you also check for it's position.